package.json

```diff
@@ -201,8 +201,12 @@
     "test":
       "mocha \"test/helper.js\" \"src/**/*.test.js\" --require @babel/register --require test/setup.js --exit",
     "test-watch": "yarn run test --watch --notify",
-    "test-cover": "nyc yarn run test",
-    "coverage": "yarn run test-cover && open coverage/lcov-report/index.html",
+    "test-cover": "yarn run test --coverage",
+    "coverage": "yarn run test-cover && opn coverage/lcov-report/index.html",
+    "dbcreate": "./node_modules/.bin/sequelize db:create",
+    "dbseed": "./node_modules/.bin/sequelize db:seed:all",
+    "dbmigrate": "./node_modules/.bin/sequelize db:migrate",
+    "dbdrop": "./node_modules/.bin/sequelize db:drop",
```

Comment:
Could you add colons to these scripts? They're valid within script names.

Also, I think these scripts run binaries from within the `node_modules` folder automatically, so you might be able to remove `./node_modules/.bin/`, although I'm not sure if it prioritizes a possibly globally-installed version if you just say `sequelize`.